Unity3D/Assets/RosSharp/Scripts/RosBridgeClient/RosCommuncation/ImagePublisher.cs

```diff
@@ -51,7 +51,10 @@ private void InitializeGameObject()
         {
```

Comment:
For easier traceability, could you please make a spearte pull request for the changes on ImagePublisher?
All minor changes (ImagePublisher, PoseStampedPublisher, Timer, Matrix3x3, StandardHeaderExtensions) can go into one pull request.